test/IISIntegration.FunctionalTests/HelloWorldTest.cs

```diff
@@ -22,26 +22,50 @@ public HelloWorldTests(ITestOutputHelper output) : base(output)
 
         [ConditionalTheory]
         [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
-        //[InlineData(RuntimeArchitecture.x86, ApplicationType.Portable)]
-        [InlineData(RuntimeArchitecture.x64, ApplicationType.Portable)]
-        public Task HelloWorld_IISExpress(RuntimeArchitecture architecture, ApplicationType applicationType)
+        [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
+        //[InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x86, ApplicationType.Portable)]
+        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        public Task HelloWorld_IISExpress(RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, ApplicationType applicationType)
         {
-            return HelloWorld(ServerType.IISExpress, architecture, applicationType);
+            return HelloWorld(ServerType.IISExpress, runtimeFlavor, architecture, applicationType);
         }
 
-        private async Task HelloWorld(ServerType serverType, RuntimeArchitecture architecture, ApplicationType applicationType)
+        [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
+        [FrameworkSkipCondition(RuntimeFrameworks.CLR)]
+        //[InlineData(RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Portable)]
+        // TODO reenable when https://github.com/dotnet/sdk/issues/696 is resolved
+        //[InlineData(RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
+        [InlineData(RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        public Task HelloWorld_IISExpress_CoreClr(RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, ApplicationType applicationType)
+        {
+            return HelloWorld(ServerType.IISExpress, runtimeFlavor, architecture, applicationType);
+        }
+
+        [ConditionalTheory]
+        [SkipIfEnvironmentVariableNotEnabled("IIS_VARIATIONS_ENABLED")]
+        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
+        [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
+        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        //[InlineData(RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Standalone)]
+        public Task HelloWorld_IIS(RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, ApplicationType applicationType)
```

Comment:
@Tratcher said this was never a thing and we never got around to actually making it a work. Perhaps this can be left removed.